packages/ngx/tests/jquery/index.html

```diff
@@ -0,0 +1,53 @@
+		</script>
+	<![endif]-->
+</head>
+<body>
+	<noscript>
+		<p role="alert">This website needs JavaScript to work properly.</p>
+	</noscript>
+
+	<a class="test-link" href="../">&larr; back to the module index</a>
+
+	<h1>JQuery test: ngx</h1>
+
+	code here
```

Comment:
since this is a angular only package it probably doesn't need jquery entry files?